src/handler.ts

```diff
@@ -80,16 +99,24 @@ function matchesFile(files: string[], fileParam: string): boolean {
  * Check for missing configurations in the Sentry extension settings
  * @param settings
  */
-export function findEmptyConfigs(settings?: SentryProject, path?: string): string[] {
-    if (!path) {
-        path = 'settings'
+export function findEmptyConfigs(settings?: SentryProject, path?: string, index?: number): string[] {
+    if (!settings && !path && !index) {
+        return ['settings']
+    }
+
+    if (path === 'project' && typeof index === 'number') {
+        path += '[' + index + ']'
     }
-    if (!settings) {
+    // Check object key length to safeguard against user error of setting an empty repositories array
+    if ((!settings || Object.keys(settings).length === 0) && path) {
         return [path]
     }
 
+    // Add repositories to missingConfigs array when no settings.filters are set and return
     let missingConfigurations: string[] = []
-
+    if (settings instanceof Object && 'projectId' in settings && !settings.filters) {
+        return missingConfigurations.concat(path + '[' + index + '].filters[0].repositories')
+    }
```

Comment:
- Update the docstring
- I'm pretty confused about how settings, path, and index relate to one another. Could this be expressed as JSON Schema instead?

Reply:
They relate in a way that they help inform the user which project and which filter needs to configured in case there are missing configs. E.g. the user will be prompted to add missing configurations for: `project[3].filters[2].repositories` . So when the user has, let's say, 16 different projects setup, they will know right away which project needs completion in their settings.